### ISSUE TITLE
Missing Paste as Plain Text hotkey in a11y instructions

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor-dev#413]
## What type of issue would you like to report?

Feature request

## [Check if the issue is already reported](http://docs.ckeditor.com/#!/guide/dev_issues_readme-section-avoid-duplicates)

http://dev.ckeditor.com/ticket/16980

## Provide detailed reproduction steps (if any)

Now that we improved handling for paste as plain text, it would make sense to add a proper information in Accessibility Instructions dialog (alt+0).

So we want to put hotkey for paste as plain text:
* `ctrl+shift+v` on Windows
  * For Edge it's `ctrl+shift+v` followed by `ctrl+v`
* `cmd+alt+shift+v` on Safari, Mac

To reproduce:

1. Open page with CKEditor instance.
1. Focus the editor.
1. Press `alt+0`.

### Expected result

The dialog should contain Paste as Plain Text, `ctrl+shift+v` (and proper variation on other OSes) key shortcut.

### Actual result

There's no Paste as Plain Text listed in the dialog.

**Comment by stalker780** [comment]
A button "paste as plain text" doesn't work in firefox now. It says "Your browser does not allow you to paste plain text this way. Press Ctrl+Shift+V to paste."
And shortcut ctrl+shift+v is also used by AddBlockPlus :(

So there is no way to paste plain text in CKEditor 4.7.1 :(

In 4.6 and 4.7 button works fine in Firefox.
